### ISSUE TITLE
Endpoint Error

### ISSUE THREAD

**Issue by Alex-Graham** [dmhacker/alexa-youtube-skill#70]
Hi,

I ran through your latest instructions using node 10 and I had reached entering the ARN into the AWS Lambda ARN but when I went to save the endpoint, it said saved failed with the following error message.

![image](https://user-images.githubusercontent.com/54984828/77761237-a14a5b00-702f-11ea-8ef2-e9585d6ff23d.png)

**Comment by janjaali** [comment]
You need to setup the "Alexa Skill Kit" with you Skill ID as a trigger for your lambda in the AWS-console where you have configured your lambda.

**Comment by Alex-Graham** [comment]
Ah ok, have I missed a few steps at the start then and if so can you link me to those steps?

Also is the skill ID the name of the function?

**Comment by janjaali** [comment]
No, the skill ID identifies your Skill against AWS Lambda. You may find the following link helpful: https://peterdaugaardrasmussen.com/2018/07/24/alexa-error-when-not-selecting-correct-blueprint/

**Comment by Alex-Graham** [comment]
Perfect got the trigger setup but I am now missing Amazon CloudWatch Logs. How do I go about setting this up or is it needed?

**Comment by dmhacker** [comment]
If you set up the Lambda server, it should automatically come up with Amazon CloudWatch logs. You won't really need to use them, but they are helpful for me for debugging purposes.
